### PR TITLE
Webpack setup files

### PR DESCRIPTION
**Pull request by angadthandi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "react": "^16.3.0",
+    "react-dom": "^16.3.0"
+  },
+  "devDependencies": {
+    "babel-loader": "^7.1.4",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-2": "^6.24.1",
+    "webpack": "^4.4.1",
+    "webpack-cli": "^2.0.13"
```
Comment (DaWenGe) [inline package.json]: might not need this. i thought it is only for webpack-dev-server configuration

File: package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    "url": "https://github.com/DaWenGe/InventoryManagementSystem/issues"
+  },
+  "homepage": "https://github.com/DaWenGe/InventoryManagementSystem#readme",
+  "dependencies": {
+    "react": "^16.3.0",
+    "react-dom": "^16.3.0"
+  },
+  "devDependencies": {
+    "babel-loader": "^7.1.4",
+    "babel-preset-es2015": "^6.24.1",
```
Comment (DaWenGe) [inline package.json]: es2015 is deprecated, use babel-preset-env instead

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+var DIST_DIR = path.resolve(__dirname, "dist");
+var SRC_DIR = path.resolve(__dirname, "source");
+
+var config = {
+	entry: SRC_DIR + "/js/app.js",
+	output: {
+		path: DIST_DIR + "/js",
+		filename: "bundle.js",
+		publicPath: "/js/"
```
Comment (DaWenGe) [inline webpack.config.js]: this line is for webpack-dev-server i think, might not need it

**Review by DaWenGe — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 3 above)
Comment (angadthandi) [reply webpack.config.js]: done

File: package.json (hunk 2 above)
Comment (angadthandi) [reply package.json]: done

File: package.json (hunk 1 above)
Comment (angadthandi) [reply package.json]: done

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+		filename: "bundle.js"
+	},
+	module: {
+		rules: [
+			{
+				test: /\.js?/,
+				include: SRC_DIR,
+				loader: "babel-loader",
+				query: {
+					presets: ["react", "es2015", "stage-2"]
```
Comment (DaWenGe) [inline webpack.config.js]: es2015 to env

**Review by DaWenGe — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 4 above)
Comment (angadthandi) [reply webpack.config.js]: done

**Review by DaWenGe — APPROVED** [review]

**Pull request merged**